### ISSUE TITLE
I want YOU to take over my mods

### ISSUE THREAD

**Issue by Lizard-Of-Oz** [RubixDev/Inventorio#278]
I'm looking for someone with experience to take over Inventorio and my other Minecraft mods.
I just can't find time in my schedule to maintain them.
There's also the 4th mod not listed on github that was sitting at 90% completion for a literal year.

**Comment by Entyco** [comment]
I love this mod and I hope you find someone that can keep it going!

**Comment by Enderteck** [comment]
Gonna look

**Comment by ItsYusei99** [comment]
I want to help, i want to improve in my programming skills

**Comment by Lizard-Of-Oz** [comment]
@ItsYusei99 Thanks for the offer, but I'm looking for someone with a lot of experience with Minecraft mods, because Inventorio is very complex and relies on lots of specific knowledge of modding Minecraft.

**Comment by lucascoelho07** [comment]
Sorry if this is off-topic, but considering you're busy with other things, do you still plan on updating Inventorio to support newer/future versions of the game, like 1.21?

**Comment by Lizard-Of-Oz** [comment]
> Sorry if this is off-topic, but considering you're busy with other things, do you still plan on updating Inventorio to support newer/future versions of the game, like 1.21?

It breaks my heart to say it, but, unfortunately, no.

**Comment by RubixDev** [comment]
I may be able to take over development on this mod, although I'm not quite sure yet. I do have some modding experience and am currently maintaining [EnchantedShulkers](https://modrinth.com/mod/enchantedshulkers), [Rug](https://modrinth.com/mod/rug), and [CarpetGamerules](https://modrinth.com/mod/carpetgamerules). Although those are all made in Java, I have also used Kotlin before, just not for Minecraft mods.

I'm not sure whether I'd want to maintain the Forge side of things though. I've only really developed mods for Fabric so far and am not a huge fan of Forge, so I'd probably lack motivation for that.

Also, before I commit to wanting to maintain this, I'll need to have a closer look around the current code base and get a good understanding of the structure and stuff. I only played a bit with the code yet as part of PR https://github.com/Lizard-Of-Oz/Inventorio/pull/306. But I thought I'd just let you know that I'm at least considering it, as I really like what this mod does.

**Comment by Enderteck** [comment]
Hell yeah

**Comment by RubixDev** [comment]
Alright, so I've finally had a proper look around the codebase now and I'm hereby applying for maintainer rights. I still have a couple of questions though:

- Is there any particular reason why the `me.lizardofoz.inventorio.api` package is written in Java and not Kotlin?
- do you know of any mod that actually adds custom toolbelt slots?
- am I allowed to create a Modrinth page for this mod?
- why do you have two separate mod pages on CurseForge for Fabric/Forge instead of one combined one?
- what would you imagine my role to be in case you accept my application? Would I simply have write access on this repo and be added as a project member on CurseForge to maintain the mod but you also keep all your current rights and make the decisions about code style and mod features, or would you completely transfer ownership over to me and give me complete freedom over this mod's future, or literally anything else in between those two extremes? I'm happy with anything, I'd just like to know what your ideas were
- do you have a Discord account or similar so we could communicate more easily?

**Comment by RubixDev** [comment]
For anyone else following this issue, just after I posted that comment last week we got in touch on Discord and my questions have been answered there. Since then I have been given access to the CurseForge pages and created a [Modrinth page](https://modrinth.com/mod/inventorio) :tada:. The only thing I'm still waiting on is a decision on how to handle this GitHub repo, i.e. whether ownership is transferred to me, I get write access, or I simply create a new fork. In the meantime i've already done some work on the code locally which I'll publish as soon as that decision is made.
